src/App.tsx

```diff
@@ -1,19 +1,156 @@
 import React from 'react';
-import './App.css';
+import './App.scss';
 
 import users from './api/users';
+import todos from './api/todos';
 
-const App: React.FC = () => {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
+const copyTodos = [...todos];
 
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
-    </div>
-  );
-};
+export class App extends React.Component <{}, {}> {
+  state = {
+    title: '',
+    userId: 0,
+    noUser: false,
+    noTitle: false,
+  };
 
-export default App;
+  handleChangeUser = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    const { value } = event.target;
+
+    this.setState({
+      userId: Number(value),
+      noUser: false,
+    });
+  };
+
+  handleChangeTitle = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const { name, value } = event.target;
+
+    this.setState({
+      [name]: value,
+      noTitle: false,
+    });
+  };
+
+  submitForm = () => {
+    const {
+      title, userId, noUser, noTitle,
+    } = this.state;
+
+    if (userId !== 0 && title !== '') {
+      copyTodos.push(
+        {
+          userId: +userId,
+          title,
+          id: copyTodos.length + 1,
+          completed: false,
+        },
+      );
```

Comment:
You need to store todos in component state.
Adding todo also should be handled using setState.